### ISSUE TITLE
Sliding right causes column refresh

### ISSUE THREAD

**Issue by agilob** [stefan-niedermann/nextcloud-deck#250]
**Describe the bug**
Sliding right (from left to right) triggers refresh, but sliding left moves to column on left

**Comment by desperateCoder** [comment]
Hi @agilob, thanks for your report!

Are you sure, you did not accidentally swipe down a little? I can't really reproduce it. Maybe a video of the behavior could help, too.

**Comment by agilob** [comment]
Hmm seems to work today, I tried it like 10 times before reporting the issue and I couldnt swipe-right, it was always refreshing widgets and loading circle was showing. Works fine today, sorry.

**Comment by desperateCoder** [comment]
😄 no problem buddy, I was just wondering if we messed it up at our last release or something

**Comment by vadiof** [comment]
I faced this issue on 0.0.18 and it is still exist on 0.1.0 from F-droid. Looks like some input handler is absolutely intolerant to down swipe. I've uploaded video of this flaw.
![ScreenRec-20200306-052009](https://user-images.githubusercontent.com/29149307/76044787-41422680-5f53-11ea-870c-973366ee624f.gif)
Also I would suggest devs have a look how similar input situation handles in SkyTube, for exmple. It is smooth and tolerant for little irregularities of swiping
